filters/redirect.go

```diff
@@ -16,21 +15,21 @@ type Redirect struct {
 
 func (spec *Redirect) Name() string { return "redirect" }
 
-func (spec *Redirect) MakeFilter(id string, c skipper.FilterConfig) (skipper.Filter, error) {
-	invalidArgs := func() (skipper.Filter, error) {
+func (spec *Redirect) MakeFilter(id string, config []interface{}) (Filter, error) {
+	invalidArgs := func() (Filter, error) {
 		return nil, errors.New("invalid arguments")
 	}
 
-	if len(c) != 2 {
+	if len(config) != 2 {
 		return invalidArgs()
 	}
 
-	code, ok := c[0].(float64)
+	code, ok := config[0].(float64)
 	if !ok {
 		return invalidArgs()
 	}
```

Comment:
i think in this case we can save that code and leave to the audience whether they want to do something irregular.